modules/data-table/main/index.coffee

```diff
@@ -746,22 +746,21 @@ class DataTable extends hx.EventEmitter
               @_.sortColPicker.value({value: currentSort.column + currentSort.direction})
 
           # populate the page size picker if there are options set
-          if filteredCount isnt undefined and filteredCount > 0
```

Comment:
remove this if as the filteredCount seems unrelated to the block it is surrounding.
The problem with this is that it makes the pageSizeOptions always get rendered without any items